bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/StatisticYear.java

```diff
@@ -447,7 +439,7 @@ public void updateStatistic(final StatisticContext statContext) {
       reorderStatisticData();
 
       // reset min/max values
-      if (_isSynchScaleEnabled == false && statContext.isRefreshData) {
```

Comment:
@wolfgang-ch I think that was modified per this new [4.20 cleanup](https://www.eclipse.org/eclipse/news/4.20/jdt.php#redundant-truth) that is checked by default, let me know if you don't like it and you want me to revert it

Reply:
This new default is just awful, I just compared 2021-03 with 2021-06, I will also check the other 2 new defaults

![image](https://user-images.githubusercontent.com/1283445/123206782-942f4c80-d4bc-11eb-96e5-34d9fff74e69.png)

Reply:
It seems that Eclipse is adding clean-ups that can be problematic for MT. As long as it's disabled in the pref file, we should be good

Reply:
Every new Eclipse version has suprises, we have to check when using new Eclipse versions, if there are new clean-up defaults. The last new Eclipse versions had a ton of new clean-up options, so I skiped reading all of them in the news notes.

Reply:
I've documented the current clean-up settings in https://github.com/wolfgang-ch/mytourbook/blob/2e0b03acc6d3380f0edd796fa93382cab4594489/bundles/net.tourbook/.settings/clean-up-setting-2021-06.png so we can easily compare it